nomenclature/definitions/variable/emissions/emissions.yaml

```diff
@@ -290,6 +290,42 @@ Emissions|CO2|AFOLU:
                 (IPCC category 3)
    unit: Mt CO2/yr
 
+Emissions|CO2|Electricity:
+   description: CO2 emmissions of power plants .
+   unit: tons/MWh
```

Comment:
I am skeptical to have this variable as "per unit of generated electricity", because other `Emissions|CO2|*` variables are reporting **total emissions** - so this could easily lead to confusion.

If you want to add a variable to describe operational characteristics, can we find an alternative name?

@sandrinecharousset what do you think?

Reply:
For a power plant we can use emission rate

Reply:
@danielhuppmann @arght I agree with the proposal of using emission rate instead of emission; I will include it in 'review mode' so that erik can accept changes easily (those are variables I did create in an older PR)

Reply:
Ok, Thanks @sandrinecharousset